src/app/core/ui/latest-changes/changelog/changelog.component.ts

```diff
@@ -68,7 +80,7 @@ export class ChangelogComponent implements OnInit {
         .subscribe((changelog) => (this.changelogs = changelog));
     }
 
-    this.customizeMarkdownRenderer();
+    // this.customizeMarkdownRenderer();
```

Comment:
we are loosing special color formating for the changelogs here ...

Reply:
@tomwwinter : any reason to disable this? When I uncomment it, it seems to work fine